<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>9</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>设备</string>
    </property>
    <widget class="QPushButton" name="pushButton_search">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_connect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_device">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_device_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>设备信息</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_rx">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_tx">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>401</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>操作</string>
    </property>
    <widget class="QPushButton" name="pushButton_clean_rx">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>清空接收</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clean_tx">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>清空发送</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_auto_new_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>自动换行</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_show_hex">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>16进制显示</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_hex_send">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>16进制发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>311</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
